prepare documentation for final deliverables

sarahwalker10 opened the pull request.

Commit:
prepare documentation for final deliverables

Commit:
formatting changes

Commit:
minor tweaks

Commit:
making directions more detailed

chiertu committed:
make GenerateSingleSourceofTruth.py runnable

chiertu committed:
reformatting to pass pre-commit

chiertu committed:
delete one extra line

chiertu committed:
add that one extra line back bc pre-commit

chiertu committed:
make GenerateDistMatrix.py runnable, add doc string

chiertu committed:
make GenerateDistMatrix.py runnable, add doc string

chiertu committed:
add CreateFullSericeDataframe.py and update README

chiertu committed:
reformat to pass pre-commit checks

chiertu committed:
update README

chiertu committed:
reformat contributors section in README

chiertu commented:
So far I have checked these two scripts, making sure they run and output files at the correct places, and have doc strings:
1. CreateFullServiceDataframe.py
2. GenerateDistMatrix.py

I decided to archive GenerateSingleSourceofTruth.py because:
1. it does the same thing as CreateFullServiceDataframe.py
2. CreateFullServiceDataframe.py is better connected with downstream data processing

chiertu committed:
archive GenerateSingleSourceofTruth.py

HuanlinDai committed:
adding route.viz section in config_inputs.ini because it keeps disappearing (last seen in commit
1f47766 in SarahPrepareDeliverables branch; editing readme to be a bit more informative

chiertu committed:
merge RouteVizConfig into branch

chiertu committed:
bug fix in bike_conversion.py

chiertu commented:
1. Bike aggregation points assignment has erroneous results. Fixed that with line "key = bike_no_agg_ind[int(col)]". This change will cause changes in map and csv outputs, and possibly routing results.
2. Comments at the end of the line was causing issue in config_inputs.ini. Fixed by moving comments into their own lines.

chiertu committed:
put args in one place in optimize_cvrp.py

chiertu committed:
reformat for pre-commit checks

chiertu committed:
fix bug in complete_feasibility_report.py, not fully done

chiertu committed:
reformat to pass pre-commit checks

chiertu commented:
Some final comments:
1. Combining sections of configuration files would eliminate lots of manual input. Ex. [convert.bike] and [optimize google cvrp]
2. There was a bug in the bike assignment that could potentially change the routing results. I think we should rerun the optimization and generate new output files
